tests/features/open_test.py

```diff
@@ -35,6 +35,55 @@ def test_open(run, tmpdir):
 
         h.press('^X')
         h.await_text('hello world')
+        h.press('^X')
+        h.await_exit()
+
+
+def test_file_glob(run, tmpdir):
+    base = 'globtest'
+    prefix = base + 'ffff.txt'
```

Comment:
rather than variables I'd rather see the actual strings in the assertions -- rather than having to hunt around for what is actually being tested the expressions should be clear in tests. https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html